src/lib/ui/Socket_Connection.svelte

```diff
@@ -3,10 +3,13 @@
 
 	import type {Socket_Store} from '$lib/ui/socket_store.js';
 
-	export let url = 'ws://localhost:3000/ws';
+	let url = `ws://localhost:3002/ws`;
 	export let socket: Socket_Store;
 
 	onMount(() => {
+		let hostname = window.location.hostname;
```

Comment:
```suggestion
		const {hostname} = window.location;
```
simple change to prefer `const` and destructuring